CMakeLists.txt

```diff
@@ -1818,16 +1818,25 @@ if(NOT CMAKE_C_COMPILER_ID STREQUAL "ARMClang")
   )
 endif()
 
-# Generate and use MCUboot related artifacts as needed.
-if(CONFIG_BOOTLOADER_MCUBOOT)
+# Generate signed (MCUboot or other) related artifacts as needed. Priority is:
+# * Sysbuild (if set)
+# * SIGNING_SCRIPT target property (if set)
+# * MCUboot signing script (if MCUboot is enabled)
+#
+# The resultant signing script is not set to the SIGNING_SCRIPT target property so that it will
+# not remain sticky if e.g. an application is configured with MCUboot then MCUboot is disabled
+# by using menuconfig.
```

Comment:
This explanation is odd, because target properties don't persist between runs.

Reply:
Removed